scripts/west_commands/zephyr_ext_common.py

```diff
@@ -23,6 +26,30 @@
 ZEPHYR_SCRIPTS = ZEPHYR_BASE / 'scripts'
 ZEPHYR_CMAKE = ZEPHYR_BASE / 'cmake'
 
+def get_twister_path():
+    """Get twister path using based on the following hierarchy
+    1) ZEPHYR_BASE env variable is set
+    2) twister.path exists in west manifest
+    3) Based on current path of this file.
+    """
+
+    zephyr_base = os.environ.get("ZEPHYR_BASE")
+
+    if zephyr_base is None:
+        try:
+            topdir = Path(util.west_topdir(start=Path.cwd(), fall_back=True))
+            config = Configuration(topdir=topdir)
+            twister_path = config.get("twister.path", str(ZEPHYR_SCRIPTS))
```

Comment:
I don't think self.config is safe for me to use in `do_add_parser`.  I've tried creating an `_import_twister` function to handle the dynamic loading of twister to use self.config in `do_add_parser` but I was getting errors.  I don't consider myself as a python expert, so if you have any tips for me here, please let me know :smiley:.  You may have noticed that I used `west_commands/build.py` as a template for my changes.

Reply:
The more I think about it, the less I like the idea of making this configurable. The twister_cmd.py and twisterlib implementations are in the same repository, and they're coupled. Getting twister_cmd.py from zephyr repository A and twisterlib from repository B would be like getting `west build` from repository A and the build system from repository B in some sense -- it just doesn't make sense to me and I think the case analysis by application type shows it's not necessary.

Reply:
... you still haven't addressed this; why are you asking for a new review?

Reply:
Sorry - missed this request.  Removed west config for path.  Now it will check `ZEPHYR_BASE` and if that doesn't exist will base it off current path of `zephyr_ext_common.py`